webpack.config.js

```diff
@@ -36,7 +36,7 @@ function getPerformanceHints(size) {
 const config = {
     devServer: {
         https: true,
-        host: '0.0.0.0',
+        host: '127.0.0.1',
```

Comment:
Why is this necessary? It helps test from multiple machines.

Reply:
Because 0.0.0.0 is causing following error on the AOT window when debugging with the jitsi-electron app:
Uncaught SyntaxError: The URL 'about:0.0.0.0:8080/sockjs-node' is invalid
    at new SockJS (main.js:70)
    at new SockJSClient (SockJSClient.js:43)
    at initSocket (socket.js:20)
    at Object.<anonymous> (index.js:176)
    at Object../node_modules/webpack-dev-server/client/index.js?https://0.0.0.0:8080 (alwaysontop.min.js:45855)
    at __webpack_require__ (bootstrap:19)
    at Object.0 (index.js:3)
    at __webpack_require__ (bootstrap:19)
    at bootstrap:83
    at bootstrap:83


I could revert and set this only locally when I need to when debugging AOT.

Did not see anything obviously wrong after setting to this new IP though.

Reply:
Got it, thanks for investigating that one!